File: programers_Lv2/숫자의 표현.js
function solution(n) {
  var answer = 0;
  for (let i = 1; i < n + 1; i++) {
    let temp = 0;
    for (let j = i; j < n + 1; j++) {
      if (temp < n + 1) {
        temp += j;
      }
      if (temp == n) {
        answer++;
        continue;
      }
      if (temp > n) {
        break;
      }
    }
  }
  return answer;
}

console.log(solution(15));

File: programers_Lv2/숫자의 표현.py
def solution(n):
    answer = 0
    for i in range(1, n+1):
        temp = 0
        for j in range(i, n+1):
            if temp < n+1:
                temp += j
            if temp == n:
                answer += 1
            if temp > n:
                break # 효율성 테스트 통과 위해

    return answer

print(solution(15))